src/simularium/RemoteConverter.ts

```diff
@@ -0,0 +1,78 @@
+import { IConverter } from "./IConverter";
+import { ILogger } from "js-logger";
+import { FrontEndError, ErrorLevel } from "./FrontEndError";
+import { NetMessageEnum, WebsocketClient } from "./WebsocketClient";
+
+import jsLogger from "js-logger";
+
+export class RemoteConverter implements IConverter {
+    protected logger: ILogger;
+    public loadFile: (NetMessage) => void;
+    public handleError: (error: FrontEndError) => void | (() => void);
+    private webSocketClient: WebsocketClient;
+
+    public constructor(
+        webSocketClient: WebsocketClient,
```

Comment:
it could be very interesting to create an entirely separate websocket client instance that connects to the same server! (rather than sharing the same connection...)  We might get a bit more asynchronous/parallelism by doing that and it could keep the message passing cleaner.  I'm not sure if there would be any significant overhead.

Reply:
I agree with this, the code to set up the simulator and the convertor seems repetitive as it is now

Reply:
Alrighty! I'm not sure if there's any significant overhead to this change either, but I'll give it a whirl and we'll see how it goes